ui/src/app/components/search-input.tsx

```diff
@@ -205,11 +220,12 @@ export class SearchInput extends React.Component<SearchInputProps, SearchInputSt
             disabled={!this.props.enabled}/>
         </TooltipTrigger>
         {this._suggest() &&
-          <div data-test-id='drop-down'
+          <div data-test-id='search-input-drop-down'
             style={{...styles.dropdownMenu, ...styles.open, minWidth: '90%'}}>
             {this.state.matches.map((match, j) => {
               return (
-                <div key={j} style={this.state.hover[j] ? styles.boxHover : styles.box}
+                <div data-test-id={`search-input-dropdown-element-${j}`}
```

Comment:
should this match the naming above?
```suggestion
                <div data-test-id={`search-input-drop-down-element-${j}`}
```

Reply:
Fixed